web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -377,6 +377,29 @@ class CVEAudit extends React.Component<Props, State> {
               }}
             />
           </Table>
+          <p>
+          <AsyncButton
+            id="bootstrap-btn"
+            defaultType="btn-link"
+            icon="fa-external-link"
+            text={t("SUSE Security CVE link")}
+            action={() => window.open("https://www.suse.com/security/cve/CVE-" +
```

Comment:
Is there a reason to use this logic and an `AsyncButton` instead of using a link like a few lines further down?

Reply:
The reason is that the `btn-link` provides the icon that show more clearly that is an external link. But if you think it is better to use a link I can change it

Reply:
If I'm not mistaken, we can use the same classes on links too, see a-la `subscription-matching-util.tsx`. Since we want to navigate to an address, a link is semantically better here.

Reply:
fixed